src/tests/multihost/alltests/test_proxy_rfc2307bis.py

```diff
@@ -414,16 +381,9 @@ def test_simple_deny_groups_user_cs2_grp1(self, multihost, backupsssdconf):
                 client.login(login_timeout=10, sync_multiplier=1,
                              auto_prompt_reset=False)
             time.sleep(3)
-            execute_cmd(multihost, f'cat /var/log/secure | grep "Access denied for user {user}"')
+            check_logs(multihost, user)
         execute_cmd(multihost, "> /var/log/secure")
```

Comment:
Use the common function to delete the log.

Reply:
Is there any common method to delete/truncate the /var/log/secure logs ?

Reply:
there is non but i have added one here

Reply:
you can use this,
https://github.com/SSSD/sssd/blob/469905bfabdda66555bd179eb7b6ac7b3cd875d1/src/tests/multihost/sssd/testlib/common/utils.py#L499